ftp.go

```diff
@@ -687,6 +687,32 @@ func (c *ServerConn) Quit() error {
 	return c.conn.Close()
 }
 
+// SetUTF8 issues a OPTS UTF8 ON/OFF command.
+func (c *ServerConn) SetUTF8(on bool) error {
+	var code int
+	var message string
+	var err error
+
+	if on {
+		code, message, err = c.cmd(-1, "OPTS UTF8 ON")
+	} else {
+		code, message, err = c.cmd(-1, "OPTS UTF8 OFF")
+	}
+
+	if err != nil {
+		return err
+	}
+
+	switch code {
+
+	case StatusCommandOK:
+		return nil
+
+	default:
+		return errors.New(message)
+	}
```

Comment:
nice.  And you remind me, the function need check `ServerConn.features`  first, not all server offers UTF8 option!